documentation/assets/Losses.md

```diff
@@ -0,0 +1,155 @@
+# Losses in SG
+
+SuperGradients provides multiple Loss function implementations for various tasks:
+
+    cross_entropy
+    mse
+    r_squared_loss
+    shelfnet_ohem_loss
+    shelfnet_se_loss
+    yolox_loss
+    yolox_fast_loss
+    ssd_loss
+    stdc_loss
+    bce_dice_loss
+    kd_loss
+    dice_ce_edge_loss
+
+All of the above, are just string aliases for the underlying torch.nn.Module classes, implementing the specified loss functions.
+
+##Basic Usage of Implemented Loss Functions in SG:
+
+When using configuration files, for example training using train_from_recipe (or similar, when the underlying train method that is being called is Trainer.train_from_config(...)):
+    
+
+In your `my_training_hyperparams.yaml` file:
+  ```yaml
+    ...
+    ...
+    
+    loss: yolox_loss
+    
+    criterion_params:
+      strides: [8, 16, 32]  # output strides of all yolo outputs
+      num_classes: 80
+  ```
+  
+   `criterion_params` dictionary will be unpacked to the underlying `yolox_loss` class constructor.
```

Comment:
Dont start a sentence like this. 
start here with: "Note that two parameters define the loss function... loss: defines the type of the loss and..."

Reply:
changed in https://github.com/Deci-AI/super-gradients/pull/658/commits/2d799e1723e68f36e89b1117bc8f5d8fd237f0d0